analytic_tag_dimension_enhanced/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Analytic Accounts Dimensions Enhanced",
+    "version": "13.0.1.0.0",
```

Comment:
@Saran440 Can we please include a short summary ?

Reply:
I think everything already goes to "readmer" folder. I would think this is not necessary.